Core/clim-basic/extended-streams/stream-output.lisp

```diff
@@ -344,34 +344,32 @@
           (do ((start start (car split))
                (split splits (rest split)))
               ((null split)
-               (stream-write-output stream string nil start end)
+               (stream-write-output stream string start end)
                (setf (cursor-position (stream-text-cursor stream))
                      (values (+ left-margin
                                 (stream-string-width stream string
                                                      :start start :end split
                                                      :text-style text-style))
```

Comment:
Unless I'm reading this wrong, the string width is computed here a second time. Maybe we should use the optimization to avoid the repeated computation instead of removing it.

Reply:
you are correct - line-breaks calls it at least once, I don't know what optimization you have in mind though.